share/p2p/discovery/options.go

```diff
@@ -7,6 +7,11 @@ import (
 	"github.com/libp2p/go-libp2p/core/peer"
 )
 
+const (
+	defaultAdvertiseRetryTimeout = time.Second
+	defaultDiscoveryRetryTimeout = defaultAdvertiseRetryTimeout * 60
+)
```

Comment:
Let put into values of default params struct, to make it more consistent with other values and other default params we have.